oioswift/proxy/controllers/wrapper.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2017 OpenIO SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import wraps
+try:
+    from oio.common.exceptions import ServiceBusy as ServiceBusy
+except ImportError:
+    # TODO To delete when `oio` >= 4.1
+    class ServiceBusyMock(Exception):
+        pass
+    ServiceBusy = ServiceBusyMock
+from swift.common.swob import HTTPServiceUnavailable
+
+def handle_service_busy(fnc):
```

Comment:
As this is not a controller, it should not be in a file on its own in this directory. Can you move this code to `__init__.py` or to `oioswift.utils`?